src/main/webapp/styles/main.css

```diff
@@ -1573,6 +1573,10 @@ ul {
   background-image: url('../images/icons/green/download.png');
 }
 
+.icon-download-black {
```

Comment:
main.css is not included in the bootstrap version

Reply:
I understand, but this code change was related to old UI (without bootstrap), so was part of my initial PR.
I wasn't sure if I should remove the code or not, since, for the .ftl files, both versions (bootstrap and old one) are still present.
What do you advise?

Reply:
Thank you, I'll probably remove "old" ftl files soon, and clean the new ones from the "old" styles. So please remove this for a bootstrap-ui version